test/OpenTelemetry.Tests/Metrics/MetricOverflowAttributeTestsBase.cs

```diff
@@ -129,7 +129,6 @@ public void EmitOverflowAttributeIsOnlySetWhenMaxMetricPointsIsGreaterThanOne(in
         meterProvider.ForceFlush();
 
         Assert.Single(exportedItems);
-        Assert.Equal(isEmitOverflowAttributeKeySet, exportedItems[0].AggregatorStore.EmitOverflowAttribute);
```

Comment:
This is needed. We want to verify that `EmitOverflowAttribute` was actually set to `true` for all the test cases.